pkg/sql/catalog/schematelemetry/scheduled_job_executor.go

```diff
@@ -74,7 +74,7 @@ func (s schemaTelemetryExecutor) ExecuteJob(
 	p, cleanup := cfg.PlanHookMaker(ctx, "invoke-schema-telemetry", txn.KV(), username.NodeUserName())
 	defer cleanup()
 	jr := p.(sql.PlanHookState).ExecCfg().JobRegistry
-	r := schematelemetrycontroller.CreateSchemaTelemetryJobRecord(jobs.CreatedByScheduledJobs, sj.ScheduleID())
+	r := schematelemetrycontroller.CreateSchemaTelemetryJobRecord(jobs.CreatedByScheduledJobs, int64(sj.ScheduleID()))
```

Comment:
We could push this cast down into CreateSchemaTelemetryJobRecord.

Reply:
I don't think we actually can. There's another call of `CreateSchemaTelemetryJobRecord` from a `crdb_internal` function that allows manual triggers of the job :/.

Is there a mechanism for causing scheduled jobs to run ahead of schedule? Perhaps we could replace the bespoke `crdb_internal` function with that?